src/solver.rs

```diff
@@ -32,7 +32,7 @@ pub struct CandidateBuckets {
 /// Read in an equation string and return results from the word list
 pub fn solve_equation(
     input: &str,
-    word_list: Vec<&str>,
+    word_list: &[&str],
```

Comment:
no need to pass by value if the function which it's being passed doesn't consume it (I think)